torch/utils/cpp_extension.py

```diff
@@ -37,6 +37,17 @@
 BUILD_SPLIT_CUDA = os.getenv('BUILD_SPLIT_CUDA') or (os.path.exists(os.path.join(
     TORCH_LIB_PATH, f'{CLIB_PREFIX}torch_cuda_cu{CLIB_EXT}')) and os.path.exists(os.path.join(TORCH_LIB_PATH, f'{CLIB_PREFIX}torch_cuda_cpp{CLIB_EXT}')))
 
+# The following values were taken from the following GitHub gist that
+# summarizes the minimum valid major versions of g++ for each supported
+# CUDA version: https://gist.github.com/ax3l/9489132
+CUDA_GCC_VERSIONS = {
```

Comment:
Am I right to assume, that CUDA_GCC_VERSIONS is a map of version by min/max supported version of GCC?
If so, please use 5.4 as lower boundary, and set upper boundary for CUDA-11.x compiler (can be extracted by looking at cuda_runtime.h)